electrum/plugins/coldcard/coldcard.py

```diff
@@ -608,7 +608,8 @@ def show_address(self, wallet, address, keystore: 'Coldcard_KeyStore' = None):
             pubkey_deriv_info = wallet.get_public_keys_with_deriv_info(address)
             pubkey_hexes = sorted([pk.hex() for pk in list(pubkey_deriv_info)])
             xfp_paths = []
-            for pubkey in pubkey_deriv_info:
+            for pubkey_hex in pubkey_hexes:
+                pubkey = bytes.fromhex(pubkey_hex)
```

Comment:
Nit: I believe the electrum convention is to use `bfh` instead of `bytes.fromhex`.

Great catch on this bug!

Reply:
seems `bfh=bytes.fromhex`